src/summary/patient-chart-summary.component.tsx

```diff
@@ -1,17 +1,49 @@
 import React from "react";
 import { RouteComponentProps } from "react-router";
 import styles from "./patient-chart-summary.css";
-import HistorySection from "./history/history-section.component";
-import DocumentationSection from "./documentation/documentation-section.component";
+import DimensionsCard from "./documentation/dimensions-card.component";
+import VitalsCard from "./documentation/vitals-card.component";
+import ConditionsCard from "./history/conditions-card.component";
+import AllergyCard from "./history/allergy-card.component";
+import NotesCard from "./history/notes-card.component";
+import ProgramsCard from "./history/programs/programs-card.component";
 
 export default function PatientChartSummary(props: PatientChartSummaryProps) {
+  const config = [
```

Comment:
This is just a placeholder for now. Will create a follow-up PR which utilizes esm-config.

Reply:
I think it would be very difficult to make esm-module-config able to deal with a specification like this. I suggest using the spec [defined here](https://docs.google.com/document/d/1isZZrnR6I2ffl2qu1_prjBxVuQ5TP7zU3oHY76GeUVg/edit?pli=1#heading=h.xqmj7xdo2ybl).

Reply:
I'm confused by the google doc. How do we represent widgets which will be imported?